contracts/attesters/hydra-s1/variants/HydraS1AccountboundAttester.sol

```diff
@@ -32,33 +32,33 @@ import {HydraS1AccountboundLib, HydraS1AccountboundClaim} from '../libs/HydraS1A
  *   If a user can generate an attestation of max value 100, they can also generate any attestation with value < 100.
  *   This attester generate attestations of scores
 
- * - Ticketed
- *   Each source account gets one userTicket per claim (i.e only one attestation per source account per claim)
+ * - Nullified
+ *   Each source account gets one nullifier per claim (i.e only one attestation per source account per claim)
  *   For people used to semaphore/ tornado cash people:
- *   userTicket = hash(sourceSecret, ticketIdentifier) <=> nullifierHash = hash(IdNullifier, externalNullifier)
+ *   nullifier = hash(sourceSecret, externalNullifier) <=> nullifierHash = hash(IdNullifier, externalNullifier)
  
  * - Accountbound (with cooldown period)
  *   Users can choose to delete or generate attestations to a new destination using their source account.
  *   The attestation is "Accountbound" to the source account.
- *   When deleting/ sending to a new destination, the ticket will enter a cooldown period, so it remains occasional
+ *   When deleting/ sending to a new destination, the nullifier will enter a cooldown period, so it remains occasional
  *   User will need to wait until the end of the cooldown period before being able to delete or switch destination again
- *   One can however know that the former and the new destinations were created using the same userTicket
+ *   One can however know that the former and the new destinations were created using the same nullifier
  
  * - Renewable
- *   A userTicket can actually be reused as long as the destination of the attestation remains the same
+ *   A nullifier can actually be reused as long as the destination of the attestation remains the same
  *   It enables users to renew or update their attestations
  **/
-contract HydraS1AccountboundAttester is IHydraS1AccountboundAttester, HydraS1Base, Attester {
+contract HydraS1AccountboundAttester is IHydraS1AccountboundAttester, HydraS1SimpleAttester {
   using HydraS1Lib for HydraS1ProofData;
   using HydraS1Lib for bytes;
-  using HydraS1AccountboundLib for Request;
+  using HydraS1Lib for Request;
 
-  // The deployed contract will need to be authorized to write into the Attestation registry
-  // It should get write access on collections from AUTHORIZED_COLLECTION_ID_FIRST to AUTHORIZED_COLLECTION_ID_LAST.
-  uint256 public immutable AUTHORIZED_COLLECTION_ID_FIRST;
-  uint256 public immutable AUTHORIZED_COLLECTION_ID_LAST;
+  // Mapping to store cooldown value for each groupId
+  mapping(uint256 => uint32) internal _groupIdCooldowns;
```

Comment:
Here it's a config mapping
You should add some additional place on the storage layout here if we want to add an other config mapping in the future before the nullifiers section